pagopa/io-app
# chore: [IOBP-630] Changed `bundleName` to `pspBusinessName` from the new payment flow

## Short description
This PR aims to use the `pspBusinessName` instead of the attribute `bundleName` to show the PSP name label.

## List of changes proposed in this pull request
- Changed `bundleName` to `pspBusinessName` when used in the new payment flow

## How to test
Start the payment flow, you should be able to see the correct PSP business name while selecting one
